contentcuration/contentcuration/frontend/shared/views/IconButton.vue

```diff
@@ -1,48 +0,0 @@
-<template>
-
-  <KIconButton
-    v-bind="$attrs"
-    :tooltip="text"
-    :icon="icon"
-    ariaLabel="text"
-    :color="color"
-    :size="size"
-    :class="{ 'rtl-flip': rtlFlip }"
```

Comment:
If this component was essentially a wrapper for `KIconButton`, what were the reasons for having this apply the `rtl-flip` class? It seems that it would still be important unless this was because of a deficiency in the KDS component itself that has been fixed.